### PR TITLE
export more info from `CompactionJobInfo`

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**

### CODE REVIEW

File: tests/test_event_listener.rs [hunk 1]
```diff
@@ -109,6 +121,13 @@ fn test_event_listener_basic() {
     assert_ne!(counter.compaction.load(Ordering::SeqCst), 0);
     drop(db);
     assert_eq!(counter.drop_count.load(Ordering::SeqCst), 1);
+    assert!(
+        counter.input_records.load(Ordering::SeqCst) >
```
Comment (siddontang) [inline tests/test_event_listener.rs]: why input is bigger than output here?

Comment (zhangjinpeng1987) [reply tests/test_event_listener.rs]: Because we have written duplicated keys.

Comment (siddontang) [reply tests/test_event_listener.rs]: Got it.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**